### ISSUE TITLE
[python] KL Loss

### ISSUE THREAD

**Issue by hoangtnm** [alibaba/MNN#1833]
Hi, I'm using MNN==1.1.4 and want to use [KLDivergence](https://www.tensorflow.org/api_docs/python/tf/keras/losses/KLDivergence) loss in MNN, I found that MNN implemented [KL](https://github.com/alibaba/MNN/blob/master/pymnn/src/loss.h#L12) loss with python binding, but I'm not so sure whether MNN's implementation of `kl` is `KLDivergence` in tensorflow or not? Can you clarify this? thank you

**Comment by yyfcc17** [comment]
you can refer to Loss.cpp `_KLDivergence` function.

**Comment by hoangtnm** [comment]
@yyfcc17 so you mean they are the same? thank you

**Comment by yyfcc17** [comment]
i checked tensorflow source code, our implementation has an additional reduce_mean over batches.
